vars/groups.groovy

```diff
@@ -59,15 +43,12 @@ def mentionGroup(groups, String groupName) {
 
     def mentionString = '';
 
-    for(member in groupMembers) {
-        discordID = member.value;
-
-        if(discordID?.trim()) {
-            mentionString += "${tagTemplate.replace('[id]', discordID)}, ";
-        }
-    }
-
-    // Remove trailing ', '
+     if(discordID?.trim()) {
+         mentionString += "${tagTemplate.replace('[id]', discordID)}, ";
+     }
+   
+    // Not needed anymore?
+    Remove trailing ', '
     message = message.substring(0, message.length() - 2);
```

Comment:
I suppose the `, ` at the end of the mention string might not be needed at all if this now tags a single Discord ID rather than a list of them.

Reply:
Yea